https://redhat.atlassian.net/browse/ACM-31308 separating files part 2 (#8864)
open-cluster-management/rhacm-docs: observability/grafana_labels.adoc

      - name
----

> . For example, check the `observability-rbac-query-proxy` pod logs in the `open-cluster-management-observability` namespace to verify where the label is listed. Run the following command:

Author: ```suggestion
. Check the `observability-rbac-query-proxy` pod logs in the `open-cluster-management-observability` namespace to verify where the label is listed. Run the following command:
```

+
[source,bash]
----
oc logs -n open-cluster-management-observability deployment/observability-rbac-query-proxy | grep -i "label"
----
+
The following information might be displayed in the pod log:

----
enabled managedcluster labels: <label>
----

. To remove the managed cluster labels from the _Label_ drop-down filter in Grafana, add the label name to the `ignore_labels` list.

> . For example, check the `observability-rbac-query-proxy` pod logs in the `open-cluster-management-observability` namespace to verify where the label is listed. Run the following command:

Author: ```suggestion
. Check the `observability-rbac-query-proxy` pod logs in the `open-cluster-management-observability` namespace to verify that the label is disabled. Run the following command:
```
